File: server/services/audit_modules.go
package services

// Legacy audit module constants kept here as the shared source of truth for now.
// Registry, alias, order, and stats responsibilities have been split into
// audit_registry.go, audit_alias.go, and audit_stats.go.
const (
	AuditModuleSalesOrder        = "sales-order"
	AuditModulePurchaseOrder     = "purchase-order"
	AuditModuleCustomer          = "customer"
	AuditModuleSupplier          = "supplier"
	AuditModuleEmployee          = "employee"
	AuditModuleMaterial          = "material"
	AuditModuleInventory         = "inventory"
	AuditModuleShipment          = "shipment"
	AuditModuleLogistics         = "logistics"
	AuditModulePackagingAssembly = "packaging-assembly"
	AuditModuleChangeOrder       = "change-order"
	AuditModuleBOM               = "bom"
	AuditModuleUser              = "user"
	AuditModuleUserPermission    = "user-permission"
	AuditModuleRole              = "role"
	AuditModuleProductionLine    = "production-line"
)

const (
	AuditEngineModuleTrading     = "trading"
	AuditEngineModuleFinance     = "finance"
	AuditEngineModuleEquipment   = "equipment"
	AuditEngineModuleEngineering = "engineering"
	AuditEngineModuleWarehouse   = "warehouse"
)

File: src/features/audit-timeline/data/audit-modules.ts
export const AUDIT_MODULES = {
  salesOrder: 'sales-order',
  purchaseOrder: 'purchase-order',
  customer: 'customer',
  product: 'product',
  qualityStandard: 'quality-standard',
  supplier: 'supplier',
  drilling: 'drilling',
  engineeringSpec: 'engineering-spec',
  employee: 'employee',
  outsourcePartner: 'outsource-partner',
  outsourceOrder: 'outsource-order',
  material: 'material',
  inventory: 'inventory',
  shipment: 'shipment',
  logistics: 'logistics',
  packagingAssembly: 'packaging-assembly',
  bom: 'bom',
  user: 'user',
  userPermission: 'user-permission',
  permissionPreset: 'permission-preset',
} as const

export type AuditModuleValue =
  (typeof AUDIT_MODULES)[keyof typeof AUDIT_MODULES]
